.github/workflows/build-and-publish-chart.yml

```diff
@@ -0,0 +1,98 @@
+name: build-and-publish-chart
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-publish-chart:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install Helm
+      uses: azure/setup-helm@v3
+
+
+    - name: Build Dependencies
+      run: |
+        helm dependency update charts/cloudzero-agent/
+
+    - name : Update Chart Version
```

Comment:
This looks pretty cool and i like the fact that it forces titles of PRs to follow the commit style. 

If you're OK with how things are now we can move forward with what we have because of time restraints and then revisit the `release-please` action in the next iteration of this after we leave beta?